prerendering.bs

```diff
@@ -787,6 +794,20 @@ Add {{[DelayWhilePrerendering]}} to {{Navigator/getUserMedia()}}, {{MediaDevices
   1. If [=this=]'s [=relevant global object=]'s [=Window/navigable=] is a [=prerendering navigable=], then return.
 </div>
 
+<h4 id="web-audio-patch">Web Audio API</h4>
+
+The concept, [=allowed to start=], is used in the spec, but details are not defined as implementation details.
+To restrict auto playback while prerendering, add the following note and steps in the [=AudioContext=] interface section.
+
+<p class="note">
```

Comment:
`class="note"` is for informative notes, not normative requirements. So just use a normal paragraph and say "the following requirements" instead.

Reply:
done.